es6-shim.js

```diff
@@ -593,53 +685,58 @@
   }
 
   var ArrayShims = {
-    from: function from(iterable) {
+    from: function from(items) {
```

Comment:
I wonder if it would be easier to get the `Array.from` and relevant internal changes in first, in a separate PR, and then get the `Promise` changes in afterwards?